openpype/pipeline/publish/publish_plugins.py

```diff
@@ -288,24 +288,25 @@ def staging_dir(self, instance):
         return get_instance_staging_dir(instance)
 
 
-class ExtractorColormanaged(Extractor):
-    """Extractor base for color managed image data.
-
-    Each Extractor intended to export pixel data representation
-    should inherit from this class to allow color managed data.
-    Class implements "get_colorspace_settings" and
-    "set_representation_colorspace" functions used
-    for injecting colorspace data to representation data for farther
-    integration into db document.
-
+class ColormanagedPyblishPluginMixin(object):
+    """Mixin for colormanaged plugins.
+
+    This class is used to set colorspace data to a publishing
+    representation. It contains a static method,
+    get_colorspace_settings, which returns config and
+    file rules data for the host context.
+    It also contains a method, set_representation_colorspace,
+    which sets colorspace data to the representation.
+    The allowed file extensions are listed in the allowed_ext variable.
+    he method first checks if the file extension is in
```

Comment:
```suggestion
    The method first checks if the file extension is in
```

Reply:
Sorry, just poking around the PR because the title caught my eye (clickbait) and could not comment after seeing the typo :) everything else looks good!

Reply:
Thanks @fabiaserra for the catch.